src/main/java/com/monumental/CustomPostgreSQL9Dialect.java

```diff
@@ -9,6 +9,6 @@
 public class CustomPostgreSQL9Dialect extends PostgreSQL9Dialect {
 
     public CustomPostgreSQL9Dialect() {
-        registerFunction("fts", new FTSFunction());
+        // TODO: Either use this for location searching or delete it
```

Comment:
@njd9419 I left this class in, in case you need it for location searching. We can remove it if you end up not needing it.

Reply:
Sounds good.

Reply:
Also, doesn't this mean we can change our `hibernate.dialect` in our configuration files?

Reply:
Yes although I left it as is since I wasn't sure if you would need this for PostGIS

Reply:
Thankfully PostGIS has its own dialect so we should be good.